pvw/server/enlil.py

```diff
@@ -776,6 +776,10 @@ def snap_to_view(self, plane):
             self.view.CameraPosition = [0, 7.5, 0]
             # Point Earth to the right (Z up)
             self.view.CameraViewUp = [0, 0, 1]
+        elif plane == "initial":
+            # Reset to the initial values
+            self.view.CameraPosition = [-3, 3, 3]
+            self.view.CameraViewUp = [0, 0, 1]
         else:
             raise ValueError('Invalid string, only "ecliptic" or "meridional" '
```

Comment:
It looks like this error message and the function's initial comment also need to be updated.

Reply:
:+1: Done.